### PR TITLE
Salons

### LINKED ISSUE #15: Salons - Information about the salons
Salong A & B

General information about the salons.

User Story:
As a user I want to know some general information about the salon. 

Acceptance criteria:

- Test so the salons sites show responsive and work on both desktop and mobile

Image

### PR DESCRIPTION
**Pull request by DanneCarlson**
Just added the 2 sites about salons for the cinema. Plan to add more info (if we want to) later on. But that's another card.

Edit: Today (23/01/02) I have fixed the things that were commented below. Please check before merging. I also added data on the salons.html file.

### CODE REVIEW

**Review by OlliverLar — APPROVED** [review]
Nice job Danne!

File: salonA.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    <header class="sec">
+      <div class="header-content-wrap">
+        <img src="static/images/logo.png" alt="logo" />
+        <nav class="header-nav">
+          <button class="header-nav-toggle"></button>
+          <menu class="header-menu">
+            <li>
+              <a class="header-menu-item active" href="/index.html">Hem</a>
+            </li>
+            <li><a class="header-menu-item" href="#">Om oss</a></li>
```
Comment (mkor92) [inline salonA.html]: href= needs to be linked with about-us.html

File: salonA.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  </head>
+  <body>
+    <header class="sec">
+      <div class="header-content-wrap">
+        <img src="static/images/logo.png" alt="logo" />
+        <nav class="header-nav">
+          <button class="header-nav-toggle"></button>
+          <menu class="header-menu">
+            <li>
+              <a class="header-menu-item active" href="/index.html">Hem</a>
```
Comment (mkor92) [inline salonA.html]: Class name "active" on index.html should not be active when visiting "salonA.html"

File: salonA.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+              <li class="movies-nav-element"><a href="#">Komedi</a></li>
+              <li class="movies-nav-element"><a href="#">Action</a></li>
+              <li class="movies-nav-element"><a href="#">Drama</a></li>
+            </ul>
+        </li>
+        <li class="movies-nav">
+          <ul class="movies-nav-list">
+            <li class="movies-nav-element"><a href="#">Salonger <span class="arrow-top">&#9662;</span></a></li>
+            <li class="movies-nav-element"><a href="#">Salong A</a></li>
+            <li class="movies-nav-element"><a href="#">Salong B</a></li>
```
Comment (mkor92) [inline salonA.html]: href = should link to salons.html, salonA.html and salonB.html

File: salonB.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+  </head>
+  <body>
+    <header class="sec">
+      <div class="header-content-wrap">
+        <img src="static/images/logo.png" alt="logo" />
+        <nav class="header-nav">
+          <button class="header-nav-toggle"></button>
+          <menu class="header-menu">
+            <li>
+              <a class="header-menu-item active" href="/index.html">Hem</a>
```
Comment (mkor92) [inline salonB.html]: Class "active" on index.html should not be active if you are on salonB.html

File: salonB.html [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+            </li>
+            <li><a class="header-menu-item" href="#">Om oss</a></li>
```
Comment (mkor92) [inline salonB.html]: href need to link about-us.html

File: salonB.html [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+              <li class="movies-nav-element"><a href="#">Komedi</a></li>
+              <li class="movies-nav-element"><a href="#">Action</a></li>
+              <li class="movies-nav-element"><a href="#">Drama</a></li>
+            </ul>
+        </li>
+        <li class="movies-nav">
+          <ul class="movies-nav-list">
+            <li class="movies-nav-element"><a href="#">Salonger <span class="arrow-top">&#9662;</span></a></li>
+            <li class="movies-nav-element"><a href="#">Salong A</a></li>
+            <li class="movies-nav-element"><a href="#">Salong B</a></li>
```
Comment (mkor92) [inline salonB.html]: href = should link to salons.html, salonA.html and salonB.html

File: salons.html [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+              <li class="movies-nav-element"><a href="#">Komedi</a></li>
+              <li class="movies-nav-element"><a href="#">Action</a></li>
+              <li class="movies-nav-element"><a href="#">Drama</a></li>
+            </ul>
+        </li>
+        <li class="movies-nav">
+          <ul class="movies-nav-list">
+            <li class="movies-nav-element"><a href="#">Salonger <span class="arrow-top">&#9662;</span></a></li>
+            <li class="movies-nav-element"><a href="#">Salong A</a></li>
+            <li class="movies-nav-element"><a href="#">Salong B</a></li>
```
Comment (mkor92) [inline salons.html]: href = should link to salons.html, salonA.html and salonB.html

File: salons.html [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+    <header class="sec">
+      <div class="header-content-wrap">
+        <img src="static/images/logo.png" alt="logo" />
+        <nav class="header-nav">
+          <button class="header-nav-toggle"></button>
+          <menu class="header-menu">
+            <li>
+              <a class="header-menu-item active" href="/index.html">Hem</a>
+            </li>
+            <li><a class="header-menu-item" href="#">Om oss</a></li>
```
Comment (mkor92) [inline salons.html]: link href to about-us.html

**Review by mkor92 — CHANGES_REQUESTED** [review]
Looks good overall but i commented some changes that i would appreciate if you fix. And maybe there should be only one "salon.html", not 3 html files. And change display propertie depending on which salon you click on in menu.

Me and Markus talked about this via video and decided to keep the three pages for now.

**Review by matodorovic — APPROVED** [review]
Checked locally and sites looks good. 👍

File: salonA.html (hunk 1 above)
Comment (DanneCarlson) [reply salonA.html]: Fixed.

File: salonA.html (hunk 2 above)
Comment (DanneCarlson) [reply salonA.html]: Fixed.

File: salonA.html (hunk 3 above)
Comment (DanneCarlson) [reply salonA.html]: Fixed.

File: salonB.html (hunk 4 above)
Comment (DanneCarlson) [reply salonB.html]: Fixed.

File: salonB.html (hunk 5 above)
Comment (DanneCarlson) [reply salonB.html]: Fixed.

File: salonB.html (hunk 6 above)
Comment (DanneCarlson) [reply salonB.html]: Fixed.

File: salons.html (hunk 7 above)
Comment (DanneCarlson) [reply salons.html]: Fixed.

File: salons.html (hunk 8 above)
Comment (DanneCarlson) [reply salons.html]: Fixed.

**Review by mkor92 — APPROVED** [review]
Looks good overall! Think we have to do some changes in CSS/SCSS later on.

**Pull request merged**